### DISCUSSION TITLE
gcc-11 warning could not understand version 13.00.00

### DISCUSSION THREAD

**Discussion by thecocoa** [Homebrew/discussions#3478] [Everyday usage]
If this isn't the correct forum for beta discussions, please point me in the right direction.

macOS Ventura beta (22A5311f)

`brew --version
Homebrew 3.5.6-73-ge217fd3
Homebrew/homebrew-core (git revision 32d1145de9e; last commit 2022-07-30)
Homebrew/homebrew-cask (git revision 9dde392914; last commit 2022-07-31)`

`gcc-11 --version
gcc-11: warning: could not understand version '13.00.00'
gcc-11 (Homebrew GCC 11.3.0_2) 11.3.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.`

Is there anyway to get gcc-11 to recognize version 13.00.00?

**Comment by SMillerDev** [comment]
If you have issues with the beta versions of macOS you'll have to resolve them yourself. There is no official support for it.
